src/live.js

```diff
@@ -1,37 +1,41 @@
 'use strict';
-var rc_util = require('rippled-network-crawler/src/lib/utility.js');
 var reportGraphite = require('./lib/report_graphite');
 var processCrawl = require('./lib/process_crawl');
 var sqs_u = require('./lib/sqs_util');
 var moment = require('moment');
 var Promise = require('bluebird');
 var graphite = require('graphite');
+var HbaseClient = require('crawler-hbase').Client;
 
 module.exports = function(max, queueUrl, dbUrl, graphiteUrl, log) {
   var graphiteClient = graphite.createClient(graphiteUrl);
-  var logsql = false;
-
+  var hbaseClient = new HbaseClient(dbUrl);
```

Comment:
it's outside for not calling constructor for each message/crawl. 
HbaseClient is designed to share one connection pool amongst all the instances.